modules/extensions-spring-support/src/test/resources/models/heisenberg.json

```diff
@@ -41882,7 +41882,7 @@
           "namespace": "MULE"
         }
       },
-      "minMuleVersion": "4.5.0",
+      "minMuleVersion": "4.6.0",
```

Comment:
Method getInlinePersonalInfo has min mule version 4.6.0 because of its parameter info. Parameter info has min mule version 4.6.0 because it is of type PersonalInfo. Type PersonalInfo has min mule version 4.6.0 because of its field description. Field description has min mule version 4.6.0 because it is annotated with Text. Text was introduced in Mule 4.6.0.